hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -141,4 +147,18 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
         .build());
     }
   }
+
```

Comment:
Let's discuss when we both have a minute. It should only be launched when it gets sent a ZK task.